django_x509/base/mixin.py

```diff
@@ -0,0 +1,19 @@
+from django.http import HttpResponse
+from django.shortcuts import get_object_or_404
+from django.utils.translation import ugettext_lazy as _
+
+from django_x509 import settings as app_settings
+
+
+class CrlDownloadMixin:
+    def crl_view(self, request, pk):
+        authenticated = request.user.is_authenticated
+        authenticated = authenticated() if callable(authenticated) else authenticated
+        if app_settings.CRL_PROTECTED or not authenticated:
```

Comment:
this is wrong, it should be like it was before.
By default there's no authentication, if `CRL_PROTECTED` is set to `True` then we shall check the user is authenticated.

Reply:
Ohh! Okay will undo it. :+1: